### ISSUE TITLE
Screen reader does not announce aria-live-region with role="status"

### ISSUE THREAD

**Issue by lenzls** [mozilla-mobile/fenix#27223]
### Steps to reproduce

1. Have a tab open
2. Open https://codepen.io/simondl/pen/rNvdBQR
3. Activate the TalkBack screen reader
4. Click on button `Announce A`
5. Listen for the screen reader announcement
6. Click on button `Announce B`
7. Listen for the screen reader announcement
8. Click on button `Announce C`
9. Listen for the screen reader announcement

### Expected behaviour

In steps 5, 7 and 9, the screen reader should read out the content of the now filled aria-live region.

### Actual behaviour

In step 5, the screen reader announces the content correctly.
In steps 7 and 9, no announcement of the content is made whatsoever.

### Device name

Fairphone 4

### Android version

Android 11

### Firefox release type

Firefox

### Firefox version

105.1.0 build #2015904299

### Additional information

I've tested the same behaviour also in other environments:

- Android; chrome (version 105.0.5195.136); TalkBack screen reader
All three live regions get announced as expected.

- Desktop Windows; chrome (verison 103.0.5060.134); NVDA screen reader
All three live regions get announced as expected.

- Desktop Windows; Firefox (version 103.0); NVDA screen reader
All three live regions get announced as expected.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21695)

**Comment by mcarare** [comment]
@eeejay Is this a known issue?

**Comment by csadilek** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1810596

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
